src/Surfnet/Stepup/Identity/Value/OnPremiseVettingType.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Copyright 2021 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Surfnet\Stepup\Identity\Value;
+
+class OnPremiseVettingType extends VettingType
```

Comment:
Why not make the VettingType an interface to get rid of the inheritance?

Reply:
Mainly out of convenience: the type and __toString methods would always be the same.